routes/web.php

```diff
@@ -34,17 +38,14 @@
 })->name('verification');
 
 Route::middleware(['auth', 'log'])->group(function () {
-    Route::get('/home', 'HomeController@index')->name('home');
-    Route::get('/user', 'UserController@index')->name('user');
     Route::get('/test_mails/{mail}/{send?}', 'EmailController@testEmail');
 });
 
-Route::middleware(['auth', 'verified'])->group(function () {
-    Route::post('userdata/update_password', 'UserController@updatePassword')->name('userdata.update_password');
-});
-
 Route::middleware(['auth', 'log', 'verified'])->group(function () {
+    Route::get('/home', 'HomeController@index')->name('home');
     Route::get('userdata', 'UserController@showData')->name('userdata');
+    Route::get('/user', 'UserController@index')->name('user');
```

Comment:
Moving user page under verified middleware will bring us back to the issue that unverified users can't log out. An easy solution would be to place a logout button on `auth.verification` blade.

Reply:
Right, I thought that's not that important. Added it now, but decided it's good there for unverified users as well, easier to access.